domain/application.ts

```diff
@@ -1,8 +1,13 @@
 import { Resident } from "./resident";
 
+export interface ApplicationList {
+  results: Array<Application>;
+}
+
 export interface Application {
   id: string;
   applicant: Resident;
+  otherMembers: Array<Resident>;
```

Comment:
@nathandavies91 fyi, this is how I've mapped other people in the application for now.